automatic/utorrent/tools/ChocolateyUninstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$arguments = @{
+    packageName    = 'uTorrent'
+    installerType  = 'exe'
+    file           = Join-Path $env:AppData 'uTorrent\uTorrent.exe'
```

Comment:
Is it always guaranteed that utorrent is installed to the appdata dir?
If not, update by using the path aquired from a registry key instead (see the template generated by `choco new`)